spec/invertedSpec.js

```diff
@@ -0,0 +1,345 @@
+/* global InvertedIndex */
+/* global localStorage */
+
+const newIndex = new InvertedIndex();
+
+const testFile = [
+  {
+    title: 'Alice in Wonderland',
+    text: 'Alice falls into a rabbit hole and enters a world full of imagination.',
+  },
+
+  {
+    title: 'The Lord of the Rings: The Fellowship of the Ring.',
+    text: 'An unusual alliance of man, elf, dwarf, wizard and hobbit seek to destroy a powerful ring.',
+  },
+
+  {
+    title: 'The Lord of the Rings: The Fellowship of the Ring.',
+    text: 'An unusual alliance of man, elf, dwarf, wizard and hobbit seek to destroy a powerful ring.',
+  },
+];
+
+const testFile2 = [
+  {
+    title: 'Alice in Wonderland',
+    text: 'Alice falls into a rabbit hole and enters a world full of imagination.',
+  },
+
+  {
+    title: 'The Lord of the Rings: The Fellowship of the Ring.',
+    text: 'An unusual alliance of man, elf, dwarf, wizard and hobbit seek to destroy a powerful ring.',
+  },
+
+  {
+    title: 'The Lord of the Rings.',
+    text: 'An unusual alliance of man, elf, dwarf, wizard and hobbit seek to destroy a powerful ring.',
+  },
+];
+
+const testFile3 = [
+  {
+    title: 'Alice in Wonderland',
+    text: 'Alice falls into a rabbit hole and enters a world full of imagination.',
+  },
+
+  {
+    title: 'The Lord of the Rings: The Fellowship of the Ring.',
+    text: 'An unusual alliance of man, elf, dwarf, wizard and hobbit seek to destroy a powerful ring.',
+  },
+
+  {
+    title: 'The Lord of the Rings: The Fellowship of the Ring.',
+    text: 'Hello, is it me you are looking for?',
+  },
+];
+
+const shortFile = [
+  {
+    title: 'Alice in Wonderland',
+    text: 'Alice. falls- into@ a+ rabbit hole.',
+  },
+];
+
+const shortFile2 = [
+  {
+    title: 'Alice in Wonderland',
+    text: 'Alice. falls- into@ a+ rabbit hole.',
+  },
+
+  {
+    title: 'The Lord of the Rings: The Fellowship of the Ring.',
+    text: 'An unusual alliance of man, elf.',
+  },
+];
+
+
+const invalidTestFile = [
+  {
+    name: 'Alice in Wonderland',
+    text: 'Alice falls into a rabbit hole and enters a world full of imagination.',
+  },
+
+  {
+    title: 'The Lord of the Rings: The Fellowship of the Ring.',
+    text: 'An unusual alliance of man, elf, dwarf, wizard and hobbit seek to destroy a powerful ring.',
+  },
+
+  {
+    title: 'The Lord of the Rings: The Fellowship of the Ring.',
+    text: 'An unusual alliance of man, elf, dwarf, wizard and hobbit seek to destroy a powerful ring.',
+  },
+];
+
```

Comment:
Moving the mocked JSON data to a separate file and importing them into your test will make your code more modularized